addons/account/tests/test_sequence_mixin.py

```diff
@@ -637,6 +637,7 @@ def _flushing_compute_name(self):
 
         for move in payments.move_id:
             self.assertRecordValues(move.line_ids, [{'move_name': move.name}] * len(move.line_ids))
+            self.assertRecordValues(move.line_ids, [{'name': "Manual"}] * len(move.line_ids))
```

Comment:
manual everywhere seems dumb... 

need to check the use case with a PO. Should we change 'Manual' -> 'Manual Payment' ? Should we do something else in case there's no ref? :shrug: